src/main.ts

```diff
@@ -44,6 +44,6 @@ initItemEventHandlers(curatedCorpusEventEmitter, [
 
 (async () => {
   const { adminUrl, publicUrl } = await startServer(4025);
-  console.log(`🚀 Public server ready at http://localhost:4004${publicUrl}`);
-  console.log(`🚀 Admin server ready at http://localhost:4004${adminUrl}`);
+  console.log(`🚀 Public server ready at http://localhost:4025${publicUrl}`);
+  console.log(`🚀 Admin server ready at http://localhost:4025${adminUrl}`);
```

Comment:
FYI - the port numbers used to be incremented - and different from API to API - for when we needed to run two or more of these services locally to test/develop Curation Admin Tools. This is now not needed since we use Admin API via Dev.